custom/recording/contexts/RecordingProvider.js

```diff
@@ -27,32 +31,34 @@ export const RECORDING_IDLE = 'idle';
 
 export const RECORDING_TYPE_CLOUD = 'cloud';
 export const RECORDING_TYPE_CLOUD_BETA = 'cloud-beta';
```

Comment:
I am not sure that the `cloud-beta` recording type is a thing anymore, do you know? The docs specify `cloud`, `local`, `output-byte-stream`, and `rtp-tracks`

Reply:
correct - `cloud` is now what was previously preferred to at `cloud-beta`